Allure.SpecFlowPlugin/allureConfig.json

```diff
@@ -1,6 +1,11 @@
 ﻿{
   "allure": {
-    "directory": "allure-results"
+    "directory": "allure-results",
+    "links": [
+      "{link}",
```

Comment:
how is that works?

Reply:
I put description in wiki already: https://github.com/allure-framework/allure-csharp/wiki/Allure.Commons

Reply:
usage example: https://github.com/allure-framework/allure-csharp/wiki/SpecFlow-Adapter#links